src/pages/solutions/for-surveyors.astro

```diff
@@ -9,50 +10,88 @@ import { Image } from 'astro:assets';
 import accessiblePhoto from '@images/solutions/accessible.svg';
 import agreementPhoto from '@images/solutions/agreement.svg';
 import applicationDevelopmentPhoto from '@images/solutions/application-development.svg';
+import gps from '@images/solutions/gps.svg';
 
 import { stripeSections } from '@utils/page';
 
 const page: IStandardPageMetadata = {
   title: 'What we offer surveyors',
+  pillar: 'Solutions',
   section: stripeSections([
     {},
     { title: 'Solutions for surveyors' },
-    { title: 'Stuff' },
-    { title: 'More stuff' },
-    { title: 'GPS location services' },
+    { title: 'About the network' },
+    { title: 'Benefits of our network' },
+    { title: 'Plans for the future' },
   ]),
 };
 ---
 
 <Layout title={page.title}>
   <QuickLinks links={page.section} slot="quick-links" />
   <Section {...page.section[0]}>
-    <p>text</p>
+    <p>
+      We're proud to offer surveyors centimeter-level precision measurements through our TURN GPS Network. The TURN
+      Network is a high-precision global navigation satellite system (GNSS) providing real-time corrections and data for
+      post processing. Surveyors in Utah, Nevada, Idaho, and Wyoming are taking advantage of this useful service for a
+      variety of purposes including research, engineering, drone operations, and more!
+    </p>
   </Section>
   <Section {...page.section[1]}>
     <Image slot="image" src={agreementPhoto} alt="two people shaking hands in agreement" class="w-full max-w-[400px]" />
-    <p>text</p>
+    <p>
+      Surveyors have long relied on GPS for their land surveying needs. When GNSS networks came along, offering more
+      reliable, high-precision accuracy, the survey industry was one of the earliest adopters of the new technology.
+    </p>
+    <p>
+      Today, networks such as ours offer coverage across the globe. Our TURN and Reno networks provides coverage in not
+      only Utah, but Nevada, Wyoming, and Idaho. We continue to offer surveryors a reliable network and expert support.
+    </p>
```

Comment:
**issue**: this is supposed to be providing solutions to problems for surveyors. This isn't achieving that. We want to market to survyors here. Why do they want to use the network. What does it give them. This paragraph should probably be replaced.

Reply:
Understood, please publish what has been provided then I'll start looking closer on how to modify it.